change/@ni-nimble-blazor-91358d04-10a3-450e-995b-8970f1128006.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
If you think it's justified, could you file an issue against the NI JS/TS styleguide repo to discuss enabling this in the shared ruleset?

Reply:
Took longer to document than I thought it would be but an issue here https://github.com/ni/javascript-styleguide/issues/115